File: eNote/eNote.Services/Database/MusicShop.cs
using eNote.Model.Enums;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace eNote.Services.Database
{
    public class MusicShop
    {
        public int Id { get; set; }
        public string KorisnickoIme { get; set; }
        public string LozinkaHash { get; set; } 
        public string LozinkaSalt { get; set; }         

        public bool Status { get; set; }

        public int UlogaId { get; set; }
        public Uloge Uloga { get; set; }

        public int AdresaId { get; set; }
        public Adresa Adresa { get; set; }

        public string Naziv { get; set; }
        public string Email { get; set; }
        public string Telefon { get; set; } 
        public string RadnoVrijeme { get; set; }
        public byte[]? Slika { get; set; }

        public ICollection<Instrumenti> Instrumenti { get; set; } = new List<Instrumenti>();
    }
}

File: eNote/UI/enote_desktop/lib/models/music_shop.dart
import 'package:json_annotation/json_annotation.dart';

part 'music_shop.g.dart';

@JsonSerializable()
class MusicShop {
  int? id;
  String? korisnickoIme;
  String? naziv;
  String? status;
  String? email;
  String? telefon;
  String? slika;
  int? adresaId;
  String? adresa;
  int? ulogaId;
  String? uloga;

  MusicShop();

  factory MusicShop.fromJson(Map<String, dynamic> json) =>
      _$MusicShopFromJson(json);

  Map<String, dynamic> toJson() => _$MusicShopToJson(this);
}
